### PR TITLE
hs20: fix checksum for hostapd-2020-06-08-5a8b3662.tar.xz

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: @dangowrt 

Compile tested: Turris MOX, mvebu/cortexa53, OpenWrt master
Run tested: N/A

Description:

- Hash of the local file hostapd-2020-06-08-5a8b3662.tar.xz does not match (file: 243af130db0310a00b14ee5929d858264b26df321c47e509dbfe12d6b99e6e3b, requested: 1b2a4947034142587a3f81eac2ea27dd64650e77f8943d973623b47533be2393) - deleting download.

### CODE REVIEW

**Comment by BKPepe** [comment]
There is another issue:
```
Applying patch 050-use-dm_ddf-v1_3_dtd.patch
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- a/hs20/server/spp_server.c
|+++ b/hs20/server/spp_server.c
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
```

This happens also on Turris infrastructure, but I couldn't reproduce it locally. :(

**Comment by neheb** [comment]
That indicates that PKG_BUILD_DIR is not set correct on the buildbots.

**Comment by neheb** [comment]
And running make check FIXUP=1 does not change the hash...

**Comment by neheb** [comment]
http://w1.fi/hostap.git <-- 404 not found...

**Comment by neheb** [comment]
```
2021-07-20T11:20:56.2392178Z Hash of the downloaded file does not match (file: 1b2a4947034142587a3f81eac2ea27dd64650e77f8943d973623b47533be2393, requested: 243af130db0310a00b14ee5929d858264b26df321c47e509dbfe12d6b99e6e3b) - deleting download.
```

That's the opposite of what you wrote.

**Comment by neheb** [comment]
Yep. PKG_BUILD_DIR is wrong.

```
drwxr-xr-x.  2 mangix mangix 4096 Jul 23 20:24 hostapd-2020-06-08-5a8b3662
drwxr-xr-x. 13 mangix mangix 4096 Jun  8  2020 hs20-2020-06-08-5a8b3662
```

The former is empty. The latter is where all the files are.

File: net/hs20/Makefile [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_DATE:=2020-06-08
 PKG_SOURCE_VERSION:=5a8b366233f5585e68a4ffbb604fbb4a848eb325
 
 PKG_SOURCE_NAME:=hostapd
-PKG_MIRROR_HASH:=1b2a4947034142587a3f81eac2ea27dd64650e77f8943d973623b47533be2393
+PKG_MIRROR_HASH:=243af130db0310a00b14ee5929d858264b26df321c47e509dbfe12d6b99e6e3b
 PKG_MIRROR_VERSION:=$(PKG_SOURCE_DATE)-$(call version_abbrev,$(PKG_SOURCE_VERSION))
 PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_MIRROR_VERSION).tar.xz
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)/$(PKG_SOURCE_NAME)-$(PKG_MIRROR_VERSION)
```
Comment (neheb) [inline net/hs20/Makefile:15]: PKG_SOURCE_NAME to PKG_NAME

Comment (neheb) [reply net/hs20/Makefile:15]: Actually this should just be cleaned up. There's no good reason for all this hackery.

**Comment by dangowrt** [comment]
See fix here 3c7da5dc7d188f090c3a7f864ba6451561d2219b

**Comment by neheb** [comment]
As this was fixed, closing.

**Pull request closed without merging**